src/main/webapp/organization.js

```diff
@@ -301,6 +308,70 @@ class Organization {
   }
 
   convertIdsToEmails(moderators) {
-      return "placeholder";
+    return "placeholder";
+  }
+
+  createOpenHoursText(day) {
+    // organization.hoursOpen[index(1-7 for day of week)].propertyMap.
+    // fromToPairs.value[index(how many set of hours for that day )].propertyMap.from/to
+    const dayTimeArea = document.createElement("div");
+    dayTimeArea.classList.add("day-time-area");
+
+    const dayTimeText = document.createElement("p");
+    dayTimeText.textContent = `${day.propertyMap.day}: `;
+
+
+    if (day.propertyMap.isOpen === "true") {
+      let fromToString = null;
+      const numPairs = day.propertyMap.fromToPairs.value.length;
+      for (let i = 0; i < numPairs; i++) {
+        let from = this.parseTime(day.propertyMap.fromToPairs.value[i].propertyMap.from);
+        let to = this.parseTime(day.propertyMap.fromToPairs.value[i].propertyMap.to);
+        if (numPairs - 1 != i) {
+          fromToString = `${from} - ${to}, \n`;
+        }
+        else {
+          fromToString = `${from} -  ${to} \n`;
+        }
```

Comment:
Instead of duplicating the thing you want to print with a small variation, what if you did something more like:
fromToString = `${from} - ${to}`;
if (...) fromToString.append(',')
fromToString.append('\n')